src/nyc_trees/apps/event/templates/event/email_rsvps.html

```diff
@@ -33,12 +33,12 @@ <h4 class="modal-title">Event Registrants</h4>
             </div>
```

Comment:
Potentially unrelated to this PR, but I'm getting an `XMLHttpRequest` cross-origin request error on the "See all" link of this page.